web-console/knikubevirt/cluster-dashboard/cluster-dashboard.md

```diff
@@ -0,0 +1,71 @@
+# Cluster Overview Dashboard
+
+The Cluster Overview dashboard provides a birds-eye view of the cluster's current health, capacity, inventory, utilization, and activity.
+
+The Overview dashboard is the first cluster-level dashboard that appears when the Dashboards nav item is selected. Additional cluster-level dashboards are available as tabs.
+
+![Cluster Overview Dashboard Healthy](img/1-0-cluster-healthy.png)
+
+The dashboard is made of 7 cards, each of which was designed to be:
+
+1. Contextual - able to be used across products
+2. Reusable - applicable to multiple dashboards
+3. Responsive - viewable on any screen size
+
+![Cluster Overview Dashboard Unhealthy](img/1-1-cluster-unhealthy.png)
+
+The Overview dashboard is a particularly useful place to identify cluster-wide problems and begin troubleshooting alerts.
+
+## Cards
+
+### Details card
+
+![Details card](img/0-1-details.png)
+
+The **"what am I looking at?"** card that serves as a contextual anchor for the rest of the dashboard.
+
+### Health card
+
+![Health card](img/0-2-health.png)
```

Comment:
We should remove the reference to RHHI in the update example.

Reply:
Darn, good catch, will fix.

Reply:
Still needs an update in the link name :)

Reply:
🤦‍♂️, thank you, fixed for real this time, and fixed an update/upgrade mismatch.